include/SDL3/SDL_gpu.h

```diff
@@ -1547,7 +1550,7 @@ typedef struct SDL_GPUVertexBufferDescription
     Uint32 slot;                        /**< The binding slot of the vertex buffer. */
     Uint32 pitch;                       /**< The byte pitch between consecutive elements of the vertex buffer. */
     SDL_GPUVertexInputRate input_rate;  /**< Whether attribute addressing is a function of the vertex index or instance index. */
-    Uint32 instance_step_rate;          /**< The number of instances to draw using the same per-instance data before advancing in the instance buffer by one element. Ignored unless input_rate is SDL_GPU_VERTEXINPUTRATE_INSTANCE */
+    Uint32 instance_step_rate;          /**< Ignored, reserved for future use. */
```

Comment:
Can we change this to "reserved, must be 1 for now" and fail if it isn't set to 1? I don't want people to zero the struct and not explicitly set this, then have their apps fail later if/when we start using this field.